src/main/java/edu/hawaii/its/api/groupings/GroupingAdmins.java

```diff
@@ -0,0 +1,32 @@
+package edu.hawaii.its.api.groupings;
+
+import edu.hawaii.its.api.type.EmptyGroup;
+import edu.hawaii.its.api.type.Group;
+
+public class GroupingAdmins implements GroupingResult {
+
+    Group adminGroup;
```

Comment:
Remember to always label the member variables in class with public, private, or protected. By default, not putting anything will set it to public. Thanks!
- [ ] Put `private` before `Group adminGroup;